pages/CLA.md

```diff
@@ -10,9 +10,9 @@ You accept and agree to the following terms and conditions for Your present and
 
 ### Licenses
 
-jQuery Foundation projects (code, documentation, and any other materials) are released under the terms of the [MIT license](http://en.wikipedia.org/wiki/MIT_License), with the following exception:
+jQuery Foundation projects (code, documentation, and any other materials) are each released under the terms of the individual licenses as noted in each project repository, or, if no separate license is specified, under the terms of the [MIT license](https://tldrlegal.com/license/mit-license), with the following exception:
```

Comment:
"if no separate license is specified, under the terms of the MIT license"

I'm not sure that's legally possible. If there is no license, there is no license. @stevenayr can the CLA really assign a license to arbitrary projects?